src/subdomains/staking/application/services/staking-blockchain-address.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@nestjs/common';
+import { StakingBlockchainAddress } from '../../domain/entities/staking-blockchain-address.entity';
+import { StakingBlockchainAddressRepository } from '../repositories/staking-blockchain-address.repository';
+
+@Injectable()
+export class StakingBlockchainAddressService {
+  constructor(private readonly repository: StakingBlockchainAddressRepository) {}
+
+  async getAvailableAddress(): Promise<StakingBlockchainAddress> {
+    // TODO - implement proper staking address fetch
```

Comment:
@davidleomay could you kindly help with this query pls? Previously we were looking for an address via routeId, but now we only reference address in Staking itself... didn't quite get how we can get unassigned address from Staking repo. Thanks!